contracts/cosmwasm-vm/cw-ibc-core/src/ics04_channel/packet/acknowledgement.rs

```diff
@@ -352,13 +354,19 @@ impl<'a> CwIbcCoreContext<'a> {
                     let chan_end_on_a =
                         self.get_channel_end(deps.storage, port_id.clone(), channel_id.clone())?;
                     let conn_id_on_a = &chan_end_on_a.connection_hops()[0];
+
+                    let timeout_height = packet.timeout.block().unwrap();
+                    let timeout_height_str =
+                        Height::new(timeout_height.revision, timeout_height.height)
+                            .unwrap()
+                            .to_string();
```

Comment:
unwrapping timeout caused issues last time when we worked on packet timeout and we had come up with this solution:
```   let timeout_height= packet.timeout.block().and_then(|b|{
                        Some(b.height.to_string())
                    }).unwrap_or("0".to_string());```